IPC: Minors fixes

thomasfangyi opened the pull request:
Update comments and fix styles.

Patater commented:
We can't "Make token allocation/free static" because then our tests can't run. Our tests depend on being able to use the token allocator (with their own token memory, not the real box ones) in order to unit test the token allocator.

Could you please drop the commit "Make token allocation/free static"?

Thanks

thomasfangyi commented:
I've removed the commit.

In a new PR #457 I introduced a new token allocation method which probably breaks some existing tests. Any suggestions on how to resolve this?

Patater commented on api/src/ipc.c:
```diff
@@ -92,10 +92,10 @@ uint32_t ipc_allocate_token(uint32_t * tokens)
- * @return   the token, or 0 if no token available */
-void ipc_free_tokens(uint32_t * tokens, uint32_t tokens_to_free)
+ * @param    tokens_to_free[in] the tokens to free (represented by bits) */
+static void ipc_free_tokens(uint32_t * tokens, uint32_t tokens_to_free)
```
This can't be static, due to being used directly by tests.

thomasfangyi replied:
that was a miss. I'm fixing it.
